tickets/fixtures/initial_data.json

```diff
@@ -105,5 +102,5 @@
     ["add_page", "wiki", "page"],
     ["change_page", "wiki", "page"],
     ["delete_page", "wiki", "page"]]
-    }}, {"pk": 5, "model": "auth.group", "fields": {"name": "support", "permissions": [["can_change_queue", "tickets", "ticket"], ["can_change_status", "tickets", "ticket"], ["can_moderate", "tickets", "ticket"], ["can_add_moderator_only_message", "tickets", "ticketcomment"], ["add_userannotation", "tickets", "userannotation"]]}}, {"pk": 3, "model": "auth.group", "fields": {"name": "moderators", "permissions": [["change_comment", "comments", "comment"], ["delete_comment", "comments", "comment"], ["change_pack", "sounds", "pack"], ["delete_pack", "sounds", "pack"], ["change_sound", "sounds", "sound"], ["delete_sound", "sounds", "sound"], ["can_change_queue", "tickets", "ticket"], ["can_change_status", "tickets", "ticket"], ["can_moderate", "tickets", "ticket"], ["can_add_moderator_only_message", "tickets", "ticketcomment"], ["add_userannotation", "tickets", "userannotation"]]}},
+    }}, {"pk": 5, "model": "auth.group", "fields": {"name": "support", "permissions": [["can_moderate", "tickets", "ticket"], ["add_userannotation", "tickets", "userannotation"]]}}, {"pk": 3, "model": "auth.group", "fields": {"name": "moderators", "permissions": [["change_comment", "comments", "comment"], ["delete_comment", "comments", "comment"], ["change_pack", "sounds", "pack"], ["delete_pack", "sounds", "pack"], ["change_sound", "sounds", "sound"], ["delete_sound", "sounds", "sound"], ["can_moderate", "tickets", "ticket"], ["add_userannotation", "tickets", "userannotation"]]}},
```

Comment:
What's this change? From what I understand when we were looking through some tickets related to the migration/permissions problem, you can use either migrations _or_ initial_data to put things into the database. An alternative to the migration to make the group would be to have a fixture that we can load in the test. This way we wouldn't need the hack, and we also wouldn't get the output of the hack running in our tests.
I'm not worried that fixtures are reported as being "deprecated"

Reply:
Nice, I'll use a fixture for the group.
this change was to remove some unused permissions from the existing fixtures as well, although it does not really make any difference.